### ISSUE TITLE
Error message "does not have a library for the current platform."

### ISSUE THREAD

**Issue by mickael-h** [Zylann/godot_heightmap_plugin#156]
Using plugin version 1.2 with Godot version 3.2.1-stable_x11.64 on Ubuntu 18.04.
The terrain GUI won't show up and I get this error message.
I installed the plugin from the AssetLib.
[Screenshot](https://i.imgur.com/CFUousN.png)

**Comment by Zylann** [comment]
This is all what `hterrain.gdnlib` contains:
```
[general]

singleton = false
load_once = true
symbol_prefix = "godot_"
reloadable = false

[entry]

Windows.64 = "res://addons/zylann.hterrain/native/bin/win64/hterrain_native.dll"

[dependencies]

X11.64 = []
Windows.64 = []
OSX.64 = []
```
There is no way for this error to originate from my plugin, because it doesn't have this library on Linux, and it also doesn't try to load it either https://github.com/Zylann/godot_heightmap_plugin/blob/eb445038e6d952c429539be5f67b357b97973919/addons/zylann.hterrain/native/factory.gd#L6

So maybe it's just Godot epic-failing again at loading that resource to generate a thumbnail or something like that... if it prevents my plugin from activating that would be quite ironic :|

**Comment by mickael-h** [comment]
Alright. I'll try experimenting with the source code and see if I can pinpoint exactly what's causing the issue to happen.

**Comment by Zylann** [comment]
It could also be that the error is unrelated to what you are experiencing. If the plugin doesn't work, maybe another error occurred in the console.

**Comment by mickael-h** [comment]
I don't think so. It's completely isolated. It's a new project, the console only shows 1 error, and only your plugin is installed and activated.

**Comment by Wallace99** [comment]
I get that error logged to console using MacOS but the plugin works fine

**Comment by Zylann** [comment]
I'm labelling this issue as a Godot one because there is nothing the plugin can do to prevent this error from happening. Godot is probably trying to load the resource for other reasons, and fails doing so, but this should not be a failure, really. The GDNative library is just not ready yet for Linux and OSX.

It's very likely it doesn't prevent the plugin from working. If you can find the cause of your problem, try opening another issue.

**Comment by WilliamTambellini** [comment]
@Zylann  a solution would simply be to push a Linux lib hterrain_native.so the same way you build and push a .dll :  
https://github.com/Zylann/godot_heightmap_plugin/tree/master/addons/zylann.hterrain/native/bin/win64
Would you mind if I prepare a PR to do so?

**Comment by Zylann** [comment]
In fact I was thinking of setting up Github Actions to build on the available platforms, if not too hard.

**Comment by MJacred** [comment]
@Zylann: the actions are running, so this can be closed, no?
https://github.com/Zylann/godot_heightmap_plugin/actions
